Popup/Popup.module.less

```
@@ -141,33 +182,49 @@
 				.padding-start-end(.extract(@agate-popup-closebutton-title-padding, left)[], .extract(@agate-popup-closebutton-title-padding, right)[]);
 			}
 
-			&:not(.withTitle) {
-				.content {
-					.padding-start-end(.extract(@agate-popup-closebutton-content-padding, left)[], .extract(@agate-popup-closebutton-content-padding, right)[]);
-				}
-			}
-
 			&.centered {
 				.title {
-					padding: @agate-popup-closebutton-centered-title-padding;
+					.padding-start-end(.extract(@agate-popup-closebutton-title-padding, left)[], .extract(@agate-popup-closebutton-title-padding, right)[]);
 				}
 			}
 		}
 
 		.buttons {
+			height: @agate-popup-buttons-height;
 			padding: @agate-popup-buttons-padding;
+			margin-top: @agate-popup-buttons-margin-top;
 			background-color: @agate-popup-buttons-bg-color;
 			border-radius: @agate-popup-buttons-border-radius;
 		}
 
 		.content {
+			min-height: @agate-popup-content-min-height;
+			width: @agate-popup-content-width;
 			padding: @agate-popup-content-padding;
 			margin: @agate-popup-content-margin;
 		}
 
 		.closeButton {
-			.position-start-end(@agate-popup-closebutton-position-left, @agate-popup-closebutton-position-right);
-			margin: @agate-component-spacing;
+			height: @agate-popup-closebutton-height;
+			width: @agate-popup-closebutton-height;
+
+			.position(@agate-popup-closebutton-position-top, @agate-popup-closebutton-position-right, @agate-popup-closebutton-position-bottom, @agate-popup-closebutton-position-left);

```

Let's have one variable for the position.
```suggestion
			.position-start-end(@agate-popup-closebutton-position);

```